### PR TITLE
OPSEXP-2692 Build arm64 deb packages

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2692

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -166,35 +166,43 @@ jobs:
           echo "imagemagick_version=$(cat ./imagemagick-version)" >> $GITHUB_OUTPUT
           echo "imagemagick_release=$(cat ./release-version)" >> $GITHUB_OUTPUT
 
+      - name: Setup Docker Engine
+        if: matrix.target_arch == 'aarch64'
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-docker@OPSEXP-2692-setup-docker
```
Comment (gionn) [inline .github/workflows/build.yml:171]: use commit id

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -205,34 +213,41 @@ jobs:
...
```
Comment (gionn) [inline .github/workflows/build.yml:218]: use commit id

**Review by pmacius — APPROVED** [review]
looks good to me

File: .github/workflows/build.yml (hunk 1 above)
Comment (gionn) [inline .github/workflows/build.yml:171]:
```suggestion
        uses: Alfresco/alfresco-build-tools/.github/actions/setup-docker@9558268b491821a4cfce1f11d4c8f6e56adc8496 # v5.30.0
```

File: .github/workflows/build.yml (hunk 2 above)
Comment (gionn) [inline .github/workflows/build.yml:218]:
```suggestion
        uses: Alfresco/alfresco-build-tools/.github/actions/setup-docker@9558268b491821a4cfce1f11d4c8f6e56adc8496 # v5.30.0
```

**Review by pmacius — APPROVED** [review]

**Pull request merged**